setup/gcp/installer/forseti_server_installer.py

```diff
@@ -47,12 +49,22 @@ def __init__(self, **kwargs):
         """
         super(ForsetiServerInstaller, self).__init__()
         self.config = ServerConfig(**kwargs)
+        self.v1_config = None
 
     def preflight_checks(self):
         """Pre-flight checks for server instance"""
 
         super(ForsetiServerInstaller, self).preflight_checks()
-
+        _, zone, name = gcloud.get_forseti_v1_info()
+        if name is not None:
+            # v1 instance exists, ask if the user wants to port
+            # the conf/rules settings from v1.
+            self.should_migrate_v1_configs()
+        if self.migrate_from_v1:
+            self.v1_config = v1_upgrader.ForsetiV1Configuration(
+                self.project_id, name, zone)
+            self.v1_config.fetch_information_from_gcs()
+            self.populate_info_from_v1()
```

Comment:
Since this really only populating from the forseti_conf.yaml, can you please make this more clear as `self.populate_conf_info_from_v1()`?  or use `config` as you have been using that

Reply:
Updated to populate_config_info_from_v1().